data/os/RedHat/7.yaml

```diff
@@ -0,0 +1,52 @@
+---
+lookup_options:
```

Comment:
I'm not sure that these lookup_options need to be here since they are already in the `common.yaml` file.

Reply:
True those are unnecessary, I've removed them.